dist/app-sales-universityhackathon/assets/i18n/en.json

```diff
@@ -254,7 +256,33 @@
         "ai": {
             "title": "AI",
             "links": "<li> <a class=\"link\" href=\"https://www.tutorialspoint.com/artificial_intelligence/index.htm\" target=_blank>AI Tutorial</a></li><li> <a class=\"link\" href=\"http://www.philocomp.net/links/ai.htm\" target=_blank>AI References</a></li>"
+        },
+        "arduino": {
+            "title": "Arduino",
+            "links":"<li> <a class=\"link\" href =\"https://docs.arduino.cc/\" target = _blank>Arduino Documentation</a></li><li> <a class=\"link\" href =\"https://github.com/arduino/\" target = _blank>Arduino Github Resources</a></li>"
+        },
+        "external API requests ":{
+            "title": "External API Requests",
+            "links": "<li> <a class=\"link\" href =\"https://docs.xano.com/working-with-data/functions/external-api-request  \" target = _blank> What are External API Requests?</a></li><li> <a class=\"link\" href =\"https://github.com/topics/external-api  \" target = _blank>Github Resources </a></li>"
+        },
+        "security configuration ":{
+            "title": " Security Configuration",
+            "links": "<li> <a class=\"link\" href=\"https://www.sciencedirect.com/topics/computer-science/configuration-security \" target=_blank> What are Seurity Configurations? </a></li>"
+        },
+        "cloud--aws":{
+            "title":"Cloud--AWS",
```

Comment:
The titles for the cloud subsections should follow the mock: "Running on AWS", "Running on GCP", etc.